### PR TITLE
OrchardCore dotnet new templates

### PR DESCRIPTION
**Pull request by agriffard**
Templates to use with dotnet new commands :
dotnet new occms
dotnet new ocmodule 
dotnet new octheme

As discussed with @sebastienros during meeting, the corresponding packages will be built by CI and should have the version numbers, like the other modules.

### CODE REVIEW

**Comment by agriffard** [comment]
I wonder if the .template.config will be correctly included in the packages.

**Comment by agriffard** [comment]
Can it be merged to check if it builds as expected?

**Comment by sebastienros** [comment]
Check appveyor artifacts that are already built for each PR.

**Comment by agriffard** [comment]
https://ci.appveyor.com/project/alexbocharov/orchard2/build/1.0.0.6206/artifacts : The build job does not contain any artifacts.

File: src/Templates/OrchardCore.Templates.Cms.Web/OrchardCore.Templates.Cms.Web.csproj [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\OrchardCore.Build\Dependencies.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```
Comment (alexbocharov) [inline src/Templates/OrchardCore.Templates.Cms.Web/OrchardCore.Templates.Cms.Web.csproj]: Why not netcoreapp2.1?

Comment (Skrypt) [reply src/Templates/OrchardCore.Templates.Cms.Web/OrchardCore.Templates.Cms.Web.csproj]: https://github.com/OrchardCMS/OrchardCore/pull/2017

**Comment by agriffard** [comment]
OrchardCore.Templates.Cms.Web is not generated on MyGet.
@sebastienros can you remove the 1.0 version for Module and Theme on MyGet ?

**Pull request merged**